bench/sleigh/CMakeLists.txt

```
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.5)
+project(bench-sleigh)
+
+add_subdirectory("../../libs/sleigh" "sleigh")
```

We probably don't want to build all of the targets in the `sleigh` project because it's quite intensive
```suggestion
add_subdirectory("../../libs/sleigh" "sleigh" EXCLUDE_FROM_ALL)
```
To still build the spec files, you should be able to do something similar as I've done in mishegos here with `add_dependencies`
https://github.com/ekilmer/mishegos/blob/7ffd2f247efa08ce09941ce043871485f4eda8e7/src/worker/ghidra/CMakeLists.txt#L26-L28

---

[1abcecb](https://github.com/athre0z/disas-bench/pull/7/commits/1abcecb0ff0a4bf3fb475bbd1a5466f820e20fcb)